tests/unit/clients/kombu_tests.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+from baseplate.clients.kombu import KombuThriftSerializer
+from baseplate.thrift.ttypes import Loid
+from baseplate.thrift.ttypes import Request
+from baseplate.thrift.ttypes import Session
+
+
+class TestKombuThriftSerializer:
```

Comment:
since we're all pytesty now, any reason to use a class at all?

Reply:
If `kombu.py` was _only_ the serializer stuff, I wouldn't use a class.  In this case the tests file is only testing one thing but the file it's testing has several distinct things that could be tested, in which case I think it's nice to use classes just to organize the tests.  Given that, I thought it made sense to use a class here so if/when someone decides to add tests for other parts of the file, they don't have to try and figure out a way to organize the existing tests.